src/components/time_picker_slider/index.utils.ts

```
@@ -0,0 +1,32 @@
+export const getNextValue = (current: number, max: number) => {
+  return formatTimeUnit((current + 1) % max);
+};
+
+export const getPrevValue = (current: number, max: number) => {
+  return formatTimeUnit((current - 1 + max) % max);
+};
+
+export const validateHours = (hours: number | string) => {
+  const numHours = typeof hours === 'string' ? parseInt(hours, 10) : hours;
+
+  if (isNaN(numHours) || numHours < 0 || numHours >= 24) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)